translations/ja.json

```
@@ -115,12 +107,10 @@
 	"Connect to LoadBalancer services": "LoadBalancer サービスに接続します",
 	"Consider creating a cluster with larger memory size using `minikube start --memory SIZE_MB` ": "`minikube start --memory SIZE_MB` を使用して、より大きなメモリーサイズのクラスターを作成することを検討してください",
 	"Consider increasing Docker Desktop's memory size.": "Docker Desktop のメモリーサイズを増やすことを検討してください。",
-	"Container runtime must be set to \\\"containerd\\\" for rootless": "rootless の場合、コンテナーランタイムを「containerd」に設定する必要があります",
 	"Continuously listing/getting the status with optional interval duration.": "任意のインターバル時間で、継続的にステータスをリストアップ/取得します。",
 	"Control Plane could not update, try minikube delete --all --purge": "コントロールプレーンがアップデートできません。minikube delete --all --purge を試してください",
 	"Copy the specified file into minikube": "指定したファイルを minikube にコピーします",
-	"Copy the specified file into minikube, it will be saved at path \u003ctarget file absolute path\u003e in your minikube.\nDefault target node controlplane and If \u003csource node name\u003e is omitted, It will trying to copy from host.\n\nExample Command : \"minikube cp a.txt /home/docker/b.txt\" +\n                  \"minikube cp a.txt minikube-m02:/home/docker/b.txt\"\n                  \"minikube cp minikube-m01:a.txt minikube-m02:/home/docker/b.txt\"": "",
-	"Copy the specified file into minikube, it will be saved at path \u003ctarget file absolute path\u003e in your minikube.\\nExample Command : \\\"minikube cp a.txt /home/docker/b.txt\\\"\\n                  \\\"minikube cp a.txt minikube-m02:/home/docker/b.txt\\\"\\n": "指定したファイルを minikube にコピーします。ファイルは minikube 内の \u003ctarget file absolute path\u003e に保存されます。\\nコマンドの例: 「minikube cp a.txt /home/docker/b.txt」\\n              「minikube cp a.txt minikube-m02:/home/docker/b.txt」\\n",
+	"Copy the specified file into minikube, it will be saved at path \u003ctarget file absolute path\u003e in your minikube.\nDefault target node controlplane and If \u003csource node name\u003e is omitted, It will trying to copy from host.\n\nExample Command : \"minikube cp a.txt /home/docker/b.txt\" +\n                  \"minikube cp a.txt minikube-m02:/home/docker/b.txt\"\n                  \"minikube cp minikube-m01:a.txt minikube-m02:/home/docker/b.txt\"": "指定したファイルを minikube にコピーします。ファイルは minikube 内の \u003c対象ファイルの絶対パス\u003e に保存されます。\nデフォルトターゲットノードコントロールプレーンと \u003cソースノード名\u003e が省略された場合、ホストからのファイルコピーを試みます。\n\nコマンド例 : 「minikube cp a.txt /home/docker/b.txt」 +\n                  「minikube cp a.txt minikube-m02:/home/docker/b.txt」\n                  「minikube cp minikube-m01:a.txt minikube-m02:/home/docker/b.txt」",
```

```suggestion
	"Copy the specified file into minikube, it will be saved at path \u003ctarget file absolute path\u003e in your minikube.\nDefault target node controlplane and If \u003csource node name\u003e is omitted, It will trying to copy from host.\n\nExample Command : \"minikube cp a.txt /home/docker/b.txt\" +\n                  \"minikube cp a.txt minikube-m02:/home/docker/b.txt\"\n                  \"minikube cp minikube-m01:a.txt minikube-m02:/home/docker/b.txt\"": "指定したファイルを minikube にコピーします。ファイルは minikube 内の \u003c対象ファイルの絶対パス\u003e に保存されます。\nデフォルトターゲットノードコントロールプレーンと \u003cソースノード名\u003e が省略された場合、ホストからのファイルコピーを試みます。\n\nコマンド例 : 「minikube cp a.txt /home/docker/b.txt」 +\n             「minikube cp a.txt minikube-m02:/home/docker/b.txt」\n             「minikube cp minikube-m01:a.txt minikube-m02:/home/docker/b.txt」",
```
"コマンド例 : " is 13 characters when counted in half-width characters.
How about making the blank on the next lines 13 characters to align the start position?